install/dataloggers.csv

```diff
@@ -20,6 +20,7 @@ Campbell Scientific,CR6,15369,Tongariro Lower Emerald Spring,,2021-05-07T00:00:0
 Campbell Scientific,CR6,15370,Fox Glacier Kea Point,,2021-08-04T00:00:00Z,9999-01-01T00:00:00Z
 Campbell Scientific,CR6,15371,Fox Glacier Victoria Falls,,2021-11-24T11:50:00Z,9999-01-01T00:00:00Z
 Campbell Scientific,CR6,17444,Upper Tama Lake,,2022-03-30T20:40:00Z,9999-01-01T00:00:00Z
+Campbell Scientific,CR6,17445,Raoul Green Lake,,2023-01-30T02:59:59Z,9999-01-01T00:00:00Z
```

Comment:
since this site is co-located with GLKZ, I suggest to use the same Name
can you please change `Raoul Green Lake` to `Raoul Island Green Lake` ?

GLKZ entry for this is:
`Kinemetrics,Q330S/6,5440,Raoul Island Green Lake,,2021-10-24T21:28:00Z,9999-01-01T00:00:00Z`